docs/docs/troubleshooting/LIMITATIONS.md

```diff
@@ -19,5 +20,4 @@ PostgreSQL is the defined database engine for the products and cannot be modifie
 
 ### Scaling EKS
 
-You cannot change the number of the EKS cluster nodes (`desired_capacity`) and node type (`instance_types`) after provisioning the environment. 
-However, the number of application pods can be scaled up as long as the EKS cluster has enough resources.
```

Comment:
This statement is conflicting with [helm chart limitation](https://atlassian.github.io/data-center-helm-charts/troubleshooting/LIMITATIONS/#product-limitations)

Reply:
Can you name the conflict?

Reply:
Hmm I got confused with this sentence in helm chart doc _We don't support horizontal or vertical autoscaling in our products. Read about Product scaling._
I can add them back with the link as well

Reply:
That vertical and hosizental scalling refer to EKS cluster nodes. This is about product scaling.

Reply:
@jjeongatl does this need to be updated based on @nghazali comment?